Code/lib/corner/corner.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CORNER_H
+#define CORNER_H
+
+typedef enum{
+  LEFT,
+  RIGHT,
+  FORWARD,
+  BRAKE,
+}turn_direction;
+
+
+class CornerMerchantry
+{
+private:
+    bool wall_detected(float, float);
+    float *set_threshold_right;
```

Comment:
Do we need them to be pointers?

Reply:
Yep so that when they're changed elsewhere in code, it doesn't require a set method in the class again